Update table format (#14319)
dotnet/docs: docs/framework/deployment/deployment-guide-for-developers.md

> For example, to detect whether the full Japanese language pack (LCID=1041) is installed, check for the following values in the registry:

Reviewer: ```suggestion
For example, to detect whether the full Japanese language pack (LCID=1041) is installed, retrieve the following named value from the registry:
```
